### PR TITLE
Update user guide introduction

### PR DESCRIPTION
**Pull request by wenhaogoh**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -114,54 +125,102 @@ Parameter | Meaning
...
+
+Below is a table containing all the components that can be found on a budget page accompanied by their respective purposes:
+
+Component | Purpose
+--------- | -------
+Command Box | A text field for you to enter your desired commands.
+Result Display | A **scrollable** panel that displays the response messages of a command after it has been executed.
+Info Box | A panel that displays the total amount you have spent so far based on the expenditures added to the budget. It will also display the threshold of the budget if it has one.
+Title | A text field that displays the name of budget that is open.
+Expenditure Card | A card that represents an expenditure stored in a budget. It contains the name as well as the cost of the expenditure.
```
Comment (EkChinHui) [inline docs/UserGuide.md]: I think you missed out tags as well

File: docs/UserGuide.md [hunk 2]
```diff
@@ -114,54 +125,102 @@ Parameter | Meaning
...
+List View | A **scrollable** panel that can hold multiple expenditure cards.
 
 ## 4. Quick start
 (Contributed by Wen Hao)
 
+If this is your first time using NUSave, follow these simple steps to jump straight into action:
```
Comment (EkChinHui) [inline docs/UserGuide.md]: Just a nit to pick, I think it should be "jump straight into the action"

Comment (wenhaogoh) [reply docs/UserGuide.md]: Yes sir

File: docs/UserGuide.md (hunk 1 above)
Comment (wenhaogoh) [reply docs/UserGuide.md]: Thanks!

**Review by EkChinHui — APPROVED** [review]
Good commits! LGTM 💯

**Pull request merged**